composer.json

```diff
@@ -18,7 +18,7 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": ">=2.0.4",
+        "yiisoft/yii2": "*",
```

Comment:
this must keep `>=2.0.4` because it depends on the security issue fix in Json helper.